pkg/controller/controller_instance.go

```diff
@@ -513,9 +513,9 @@ func (c *controller) reconcileServiceInstance(instance *v1alpha1.ServiceInstance
 		return err
 	}
 
-	glog.V(4).Infof("Processing ServiceInstance %v/%v", instance.Namespace, instance.Name)
+	glog.V(4).Infof(`ServiceInstance "%v/%v": Processing`, instance.Namespace, instance.Name)
 
-	glog.V(4).Infof("Adding/Updating ServiceInstance %v/%v", instance.Namespace, instance.Name)
+	glog.V(4).Infof(`ServiceInstance "%v/%v": Adding/Updating`, instance.Namespace, instance.Name)
```

Comment:
Let's use the same format as I did in the other PR, and we can tweak them all later:

```
glog.V(4).Infof("ClusterServiceBroker %q: processing adding/update event", broker.Name)
```